src/components/Socials.astro

```diff
@@ -0,0 +1,36 @@
+---
+
+---
+
+<ul>
+    <li><a href="https://github.com/fujowebdev/fujowebdev/"><i class="fa-brands fa-github"></i></a></li>
+    <li><a href="https://www.kickstarter.com/projects/essential-randomness/the-fujoshi-guide-to-web-development"><i class="fa-brands fa-kickstarter"></i></a></li>
+    <li><a href="https://fujowebdev.tumblr.com/"><i class="fa-brands fa-tumblr"></i></a></li>
+    <li><a href="https://twitter.com/fujowebdev"><i class="fa-brands fa-twitter"></i></a></li>
+    <li><a href="https://blorbo.social/@fujowebdev"><i class="fa-brands fa-mastodon"></i></a></li>
+</ul>
+
+<link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.5.1/css/all.css">
```

Comment:
Ideally, we should be self-hosting these assets.

@essential-randomness do you have a preference for whether we simply host the css (slightly more complex) or if we import a package like [astro-icon](https://www.astroicon.dev/)

Reply:
I think this PR should do whatever is most in line with what we have. Personally, I'd welcome a clean up, and think it would make an excellent bug to file.